pom.xml

```diff
@@ -97,6 +97,13 @@
             <version>${jackson.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.security.oauth</groupId>
+            <artifactId>spring-security-oauth2</artifactId>
+            <version>2.0.7.RELEASE</version>
+            <scope>test</scope>
+            <optional>true</optional>
```

Comment:
According to [MNG-1378](https://issues.apache.org/jira/browse/MNG-1378):
> The problem here is that by design test dependencies are not transitive

By default they are not even installed/deployed so no one can depend on them, afaik.